tests/test_custom_fields.py

```diff
@@ -29,6 +51,16 @@ class MyContact(Contact):
     m1.save()
 
     m2 = MyContact.get_one(id=m1.id)
-    assert m2.my_field == m1.my_field
+    if field_type == FIELD_TYPE.SMART_ADDRESS:
```

Comment:
do we need this? if Entity == Entity is not working same way,  add Entity.__eq__ in requests-client